aws/resource_aws_cloudwatch_metric_alarm_test.go

```diff
@@ -209,6 +229,26 @@ resource "aws_cloudwatch_metric_alarm" "foobar" {
 }`, rInt)
 }
 
+func testAccAWSCloudWatchMetricAlarmConfigDatapointsToAlarm(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_cloudwatch_metric_alarm" "foobar" {
+  alarm_name                = "%s"
```

Comment:
Shouldn't we use the same way to name resources as other tests (e.g. for future sweepers) in this file?
(it's just me trying to keep consistency between tests as much as possible! 😄 )

Reply:
No problem about this, I'll switch it to `rInt` for consistency with the other tests.

Something I think that would be really helpful in the future is switching the testing to include the test names so we can easily find "bad apples". Just jumped the gun a little bit.

Reply:
@bflad totally agree on that :)
Was just wondering on this specific question why we were using t.Name() rather than what we do elsewhere: you may had a good reason :)